src/components/LaneCustomListsEditor.tsx

```diff
@@ -75,7 +118,12 @@ export default class LaneCustomListsEditor extends React.Component<
                             >
                               <GrabIcon />
```

Comment:
ll. 119-131 is duplicated below in ll. 188-200. Would it be useful to refactor?

Reply:
Good point. I extracted the duplicate code into its own function.